lib/cog_cmd/ecs/service/create.rb

```diff
@@ -0,0 +1,84 @@
+require 'cog_cmd/ecs/helpers'
+require 'cog_cmd/ecs/service/helpers'
+require 'securerandom'
+
+module CogCmd::Ecs::Service
+  class Create < Cog::Command
+
+    include CogCmd::Ecs::Helpers
+    include CogCmd::Ecs::Service::Helpers
+
+    attr_reader :service_name, :task_definition, :desired_count
+    attr_reader :cluster_name, :role
+
+    def initialize
+      # args
+      @service_name = request.args[0]
+      @task_definition = request.args[1]
+      @desired_count = request.args[2] || 1
```

Comment:
Might be nice to pull this "magic number" out into a `DEFAULT_DESIRED_COUNT` constant.